install.sh

```diff
@@ -42,7 +42,7 @@ nvm_source() {
 
 nvm_download() {
   if nvm_has "curl"; then
-    curl -q $*
+    curl -q "$*"
```

Comment:
double quoting this line, and the wget line below, utterly breaks this function. please revert these changes.